x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/indicator_match/threat_mapping/create_threat_signals.ts

```diff
@@ -216,6 +218,22 @@ export const createThreatSignals = async ({
     }
   };
 
+  const license = await firstValueFrom(licensing.license$);
+  const hasPlatinumLicense = license.hasAtLeast('platinum');
+  const isAlertSuppressionEnabled = Boolean(
+    completeRule.ruleParams.alertSuppression?.groupBy?.length
+  );
+
+  const isAlertSuppressionConfigured =
```

Comment:
Nit: we didn't have an explicit name for this concept before, but "configured" reads a bit odd to me here. I would expect the following definitions, which are a little mixed up here:

* configured == suppression configuration exists on the rule
* enabled == feature flag is enabled
* active == configured + enabled.

Just my 🪙 🪙 .